src/SFA.DAS.EmploymentVerification.Database/Tables/ScheduledEC.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE [dbo].[ScheduledEmploymentVerification]
+(
+  ScheduledECId BIGINT NOT NULL Identity(1,1) Constraint PK_ScheduledEmploymentVerification PRIMARY KEY,
+  CorrelationId UNIQUEIDENTIFIER NOT NULL,
+  ULN BIGINT NOT NULL,
+  UKPRN BIGINT,
+  EmployerAndProviderApprovedOn DATETIME,
+  EmployerAccountId BIGINT NOT NULL,
+  CommitmentStartDate DATETIME,
+  CommitmentId BIGINT,
+  AapprenticeshipId BIGINT,
+  CommitmentStatus SMALLINT,
+  CheckType SMALLINT,
```

Comment:
Should there be an FK for this column?

Reply:
We can add. Need your suggestion. Additionally we need Unique Key to identify unique learners and enable lookback.